Source/WebCore/page/PerformanceResourceTiming.cpp

```diff
@@ -305,12 +305,20 @@ uint64_t PerformanceResourceTiming::decodedBodySize() const
     // See https://github.com/w3c/server-timing/issues/89
     if (!m_resourceTiming.isSameOriginRequest())
         return 0;
-
+    
     auto decodedBodySize = m_resourceTiming.networkLoadMetrics().responseBodyDecodedSize;
     if (decodedBodySize == std::numeric_limits<uint64_t>::max())
         return 0;
-
+    
     return decodedBodySize;
 }
 
+uint16_t PerformanceResourceTiming::responseStatus() const
+{
+    // FIXME: This should be a check for whether CORS has failed instead of just forbidding cross-origin response statuses.
```

Comment:
Not only do test results need updating with this, but I'm not sure how to best get this bit of information to this location.